CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [3.1.0] - 2019-10-09
+
+### Added
+
+- Added support for macOS Catalina
+- Added support for Chef 15
+- Added `UserPath` and `SystemPath` libraries to simplify calls to macOS filepaths
```

Comment:
Does every line under the `Added` heading need to start with `Added`?

Reply:
I prefer full sentences grammatically, but after checking https://keepachangelog.com/en/1.0.0/, it seems you're correct.